### PR TITLE
Feature/pinch to zoom

### PR DESCRIPTION
**Pull request by marijnschilling**
Pinch to zoom, zoom is accordingly to the size of the image.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -32,3 +32,16 @@ DerivedData
...
+.idea/misc.xml
+.idea/modules.xml
+.idea/runConfigurations/iOS.xml
+.idea/runConfigurations/Tests.xml
+.idea/vcs.xml
+.idea/Viewer.iml
+.idea/workspace.xml
+.idea/xcode.xml
+Demo.xcodeproj/project.pbxproj
+Demo.xcodeproj/project.pbxproj
```
Comment (3lvis) [inline .gitignore]: 😱 This shouldn't be here

File: .gitignore [hunk 2]
```diff
@@ -32,3 +32,16 @@ DerivedData
 # CocoaPods
 Pods
 *.lock
+
+.idea/.name
+.idea/encodings.xml
+.idea/misc.xml
+.idea/modules.xml
+.idea/runConfigurations/iOS.xml
+.idea/runConfigurations/Tests.xml
```
Comment (3lvis) [inline .gitignore]: You might want to just add

```
.idea/
```

To your global gitignore so you don't have to add it to every gitignore. http://stackoverflow.com/a/22885996/717416

File: Demo.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -1,574 +0,0 @@
-// !$*UTF8*$!
-{
-	archiveVersion = 1;
-	classes = {
-	};
-	objectVersion = 46;
-	objects = {
-
```
Comment (3lvis) [inline Demo.xcodeproj/project.pbxproj:8]: gitignore removed the pbxproj.

File: .gitignore (hunk 1 above)
Comment (marijnschilling) [inline .gitignore]: Why not? I want to ignore these files don't I?

File: Demo.xcodeproj/project.pbxproj (hunk 3 above)
Comment (marijnschilling) [inline Demo.xcodeproj/project.pbxproj:8]: Yeah i thought this file should be ignored but why is it not?

File: .gitignore (hunk 2 above)
Comment (marijnschilling) [inline .gitignore]: I will do this!

**Comment by marijnschilling** [comment]
I'm really inexperienced with gitignore, we never did that at my last job 😱 🙈

File: .gitignore (hunk 1 above)
Comment (3lvis) [inline .gitignore]: `Demo.xcodeproj/project.pbxproj` is needed in order to run the project. I was only talking about that, hehe. The `.idea` ones should probably go in the global gitignore

**Comment by 3lvis** [comment]
Great work @marijnschilling! It's working quite great, I only found an issue when rotating the screen here are some steps to reproduce it:
- Rotate to from portrait to landscape
- Zoom in
- Rotate back to portrait
- Zoom back

**Comment by 3lvis** [comment]
Let's merge this PR and have a separate one for the rotation bug. What do you think?

**Comment by marijnschilling** [comment]
Good plan!

**Comment by 3lvis** [comment]
Rotation bug issue here: https://github.com/bakkenbaeck/Viewer/issues/45

**Pull request merged**